lint/check.js

```diff
@@ -68,6 +66,14 @@ log.info("-------------------------------------")
 
 function lintMeta (key) {
   const peekData = extractMeta(markdownFiles[key])
+
+  // check if the metadata is extracted properly
+  if (peekData instanceof Error && peekData.name === "YAMLException") {
+    log.error(`Invalid metadata in ${key}.\nError: ${peekData.message}`)
```

Comment:
Others generally look good.
```suggestion
    log.error(`Invalid metadata in '${key}.md'\nError: ${peekData.message}\n`)
```

Reply:
Thanks for the suggestions. :+1: